Cargo.toml

```diff
@@ -28,11 +28,12 @@ section = "utility"
 [dependencies]
 anyhow = "1.0"
 askalono = "0.4.4"
-byte-unit = "4.0.14"
+byte-unit = "=4.0.13" # to match git-repository's lower requirement, which it needs for MSRV
```

Comment:
That's a great suggestion, I wasn't really aware of that. And after trying it, it turns out that it is working exactly as you think it would, nice!

It's adjusted in 41594424e79178db1bc2b26ecba4c284aacc11df .